exopy/tasks/tasks/base_tasks.py

```diff
@@ -1172,6 +1172,9 @@ def prepare(self):
         """Optimise the database for running state and prepare children.
 
         """
+        # We cannot assume that the checks were run (in the case of a
+        # forced-enqueueing) so we need to make sure we set the default path.
+        self.write_in_database('default_path', self.default_path)
         self.database.prepare_to_run()
         super(RootTask, self).prepare()
 
```

Comment:
You could use python 3 syntax for super as some would say :) otherwise this looks reasonable

Reply:
True enough. I will make the change.